src/utils/api.ts

```diff
@@ -91,9 +91,133 @@ export const getTypeById = (
   id: paths['/types/{id}']['get']['parameters']['path']['id'],
 ) => handleResponse(instance.get(`/types/${id}`))
 
-export const getReviews = (
```

Comment:
Great data mock for demo-ing, but lets remove any changes made to this file for prod